### PR TITLE
add google() to build.gradle in CLI template

### PR DESCRIPTION
**Pull request by riwu**
## Motivation

Fixes android build issue

### CODE REVIEW

**Comment by chirag04** [comment]
@riwu what problem does this fix specifically? Can you provide a repro?

File: local-cli/templates/HelloWorld/android/build.gradle [hunk 1]
```diff
@@ -3,6 +3,7 @@
 buildscript {
     repositories {
         jcenter()
+        google()
```
Comment (chirag04) [inline local-cli/templates/HelloWorld/android/build.gradle]: also i think this only works for gradle 4 and up. RN is not using gradle 4 yet.

**Comment by riwu** [comment]
Indeed this is only a problem on newer versions of gradle.
See https://github.com/facebook/react-native/issues/15756 and [this](https://stackoverflow.com/questions/44071080/could-not-find-com-android-tools-buildgradle3-0-0-alpha1-in-circle-ci).

Is there documentation on supported versions of gradle?

**Comment by dulmandakh** [comment]
closing this PR because the same change is introduced in https://github.com/facebook/react-native/commit/6e356895e79fb92640295a14483af1a430732247

**Pull request closed without merging**